go.mod

```diff
@@ -7,14 +7,14 @@ require (
 	github.com/golang/mock v1.6.0
 	github.com/kylelemons/godebug v1.1.0
 	github.com/stretchr/testify v1.7.1
-	github.com/urfave/cli/v2 v2.8.1
+	github.com/urfave/cli/v2 v2.25.1
```

Comment:
Oh interesting I wonder where this downgrade came from

Reply:
was probably some bad merge.